Index: src/blackjack/MainGame.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package blackjack;\r\n\r\nimport javax.swing.JFrame;\r\n\r\npublic class MainGame {\r\n\r\n    public void startGame(){\r\n        //Create game window\r\n        JFrame frame = new JFrame(\"BlackJack\");\r\n        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);\r\n\r\n        //Set its size\r\n        frame.setSize(1000, 800);\r\n\r\n        //Make a new blackjack Game object/JPanel\r\n        Game blackjack = new Game();\r\n\r\n        //Add it to the frame and make it visible\r\n        frame.add(blackjack);\r\n        frame.setVisible(true);\r\n\r\n    }\r\n\r\n    public static void main(String[] args) {\r\n\r\n        MainGame test = new MainGame();\r\n        test.startGame();\r\n\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/blackjack/MainGame.java b/src/blackjack/MainGame.java
--- a/src/blackjack/MainGame.java	
+++ b/src/blackjack/MainGame.java	
@@ -23,8 +23,8 @@
 
     public static void main(String[] args) {
 
-        MainGame test = new MainGame();
-        test.startGame();
+        MainGame blackjack = new MainGame();
+        blackjack.startGame();
 
     }
 }
